client/src/components/header/header-main.js

```diff
@@ -0,0 +1,55 @@
+import Component from '@/components/component';
+
+import styles from '@/styles/components/header/header-main.module.scss';
+
+// props : { type, imageURL, onClick }
```

Comment:
```suggestion
// props : { location, onClickCategory, onClickPlace, onClickAccount, onClickMenu }
```

원래는 `placeName`이였는데 `location`으로 api와 맞추면 좋겠습니다.

Reply:
네 수정하겠습니다.